src/duration.js

```diff
@@ -236,11 +237,28 @@ export default class Duration {
    * @return {Duration}
    */
   static fromObject(obj) {
-    return new Duration({
-      values: normalizeObject(obj, Duration.normalizeUnit, true),
-      loc: Locale.fromObject(obj),
-      conversionAccuracy: obj.conversionAccuracy
-    });
+    if (obj == null || typeof obj !== 'object') {
```

Comment:
if it is undefined then `typeof obj !== 'object'`. Execution will be raised (thrown), as it should be?

Reply:
By the way `obj==null` equals `value === null || value === unefined`. See: http://st-on-it.blogspot.com/2011/03/why-null-not-null.html. In this case it's not very important to know this trick because we have `typeof obj !== 'object'`

Reply:
I'm surprised the linter allowed that, but OK with me